prometheus/exporter/gpu_exporter.go

```diff
@@ -154,6 +154,7 @@ func writeMetricsToFile(index int, logDir string) {
 func main() {
     logDir := flag.String("gpu_log_dir", "/datastorage/prom-exporters", "log written dir")   
     mode := flag.String("flag", "gpu", "gpu server")
+    sleepTime := flag.Int("sleepTime", 30, "cmd sleep duration")   
```

Comment:
>sleepTime [](start = 4, length = 9)

should also change dockerfile entry point?

Reply:
added